app/views/main.scala.html

```diff
@@ -17,8 +17,7 @@
 @import config.ApplicationConfig
 @(title: String,
   mainContentHeader: Html = HtmlFormat.empty,
-  appConfig: config.AppConfig = config.ApplicationConfig,
-  bodyClass: Option[String] = None)(mainContent: Html)(implicit request: Request[_], lang : Lang, messages: Messages)
```

Comment:
This should not have been removed, you now can't change bodyClass as its now set to None below, was this intentional?